mods/cnc/maps/gdi08b/rules.yaml

```diff
@@ -0,0 +1,180 @@
+World:
+	LuaScript:
+		Scripts: campaign-global.lua, gdi08b.lua, gdi08b-AI.lua
+	MusicPlaylist:
+		StartingMusic: march
+		VictoryMusic: gdi_win1
+	MissionData:
+		Briefing: Dr. Moebius is establishing a Hospital to treat the civilians in the region who are falling ill due to exposure to Tiberium.\n\nProtect Moebius and the civilians.\n\nEliminate Nod presence in the area.
```

Comment:
I think you should replace "Moebius" with "Mobius" because the mission is called "Protect **Mobius** In Bratislava"